README.md

```diff
@@ -8,18 +8,12 @@ Throughout this documentation, ensure that you understand the following terminol
 - Target server: new server, to which Satellite server is being cloned.
 
 #### Supported Satellite Versions ####
-- 6.6
-- 6.7
-- 6.8
-- 6.9
-- 6.10
-- 6.11
 - 6.12
 - 6.13
 
 #### What you need: ####
-  - A blank (vanilla install) RHEL 7 or RHEL 8 (Satellite 6.11+) machine (target server). You will run the setup commands here.
-  - A backup from a Satellite server (source server) created with `satellite-maintain`. This backup can be with or without pulp data, and can be from a RHEL 7 or 8 machine.
+  - A blank (vanilla install) RHEL 8 machine (target server). You will run the setup commands here.
+  - A backup from a Satellite server (source server) created with `satellite-maintain`. This backup can be with or without pulp data.
   - You will need a Satellite 6 subscription for the cloned machine. With the new Satellite Infrastructure [subscription model](https://access.redhat.com/solutions/3382781) you should have multiple Satellite subscriptions available.
 
 #### Setup ####
```

Comment:
In the instructions there's `yum install git` - should that be `dnf install git` or even `dnf install git-core`?